package.json

```diff
@@ -24,12 +24,14 @@
     "test": "jest --runInBand --forceExit --coverage",
     "test:watch": "jest --runInBand --watchAll",
     "db": "run-s db:create db:migrate",
-    "db:create": "run-p db:create:*",
+    "db:create": "run-p db:create:dev db:create:test",
     "db:create:dev": "dotenv -- sequelize db:create || exit 0",
     "db:create:test": "dotenv -e .env.test -- sequelize db:create --env test || exit 0",
-    "db:migrate": "run-p db:migrate:*",
+    "db:create:prod": "sequelize db:create --env production || exit 0",
+    "db:migrate": "run-p db:migrate:dev db:migrate:test",
     "db:migrate:dev": "dotenv -- sequelize db:migrate",
     "db:migrate:test": "dotenv -e .env.test -- sequelize db:migrate --env test",
+    "db:migrate:prod": "sequelize db:migrate --env production",
```

Comment:
How will we run this script in prod when we have changed the DB schema? I would like to have it outside the code as well, but with the restrictions we have in place we'd always have to get a special permission from the Micros team to get production credentials with write access. 

We decided to put the DB migrate code into the application because the application has all the credentials already.

The long-term solution I had in mind was to move the code into an admin endpoint so we can decouple the database migrations from a deployment.

Reply:
couple of lines ups, it runs on production start for both worker and web threads.

Reply:
Ah, missed that.

Ok. I guess we're not going to have many or risky database changes any time soon. If we do, I think we should pull it into the code again and trigger it via HTTP, so it's decoupled from startup.

Reply:
Can't say I'm a fan of that method as it'll harder to test, harder to code around, harder to maintain and would potentially be a security issue. This method of doing migrations before startup is fairly common practice and has worked well.

The only other way I would recommend it would be to completely separate DB maintenance from the app and deploy it at will. However, there's still the problem that the code is essentially having to assume that everything works fine. It also makes it harder to roll back.